packages/jest-validate/src/utils.ts

```diff
@@ -52,10 +52,14 @@ export const createDidYouMeanMessage = (
   unrecognized: string,
   allowedOptions: Array<string>,
 ): string => {
-  const suggestion = allowedOptions.find(option => {
-    const steps: number = leven(option, unrecognized);
-    return steps < 3;
+  let isBestMatch: boolean, suggestion: string;
+  
+  allowedOptions.forEach(option => {
+    isBestMatch = leven(option, unrecognized) < leven(suggestion || '', unrecognized); 
+    if (leven(option, unrecognized) < option.length * 0.4 && isBestMatch) {
+      suggestion = option;
+    } 
   });
```

Comment:
This implementation would find the best match from all of the available options.